lib/evmchain/evmchain.go

```diff
@@ -17,6 +17,9 @@ const (
 	// Testnets.
 	IDIliad uint64 = 1513
 
+	// Odyssey Testnets.
+	IDOdyssey uint64 = 1514
```

Comment:
Are we gonna use the same chain ID with mainnet for Odyssey?
I think it would better to use diff chain ID for testnet.

Reply:
We are reserving 1514 for mainnet

What about 1515?

Reply:
1516 looks good

Reply:
changed to 1516